packages/fast-components-styles-msft/src/text-action/index.ts

```diff
@@ -0,0 +1,146 @@
+    return disabledContrast(
+        designSystem.contrast,
+        foregroundNormal(designSystem),
+        designSystem.backgroundColor
+    );
+}
+
+function hoverColor(config: DesignSystem): string {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    return hoverContrast(
+        designSystem.contrast,
+        foregroundNormal(designSystem),
+        designSystem.backgroundColor
+    );
+}
+
+/* tslint:disable:max-line-length */
+const styles: ComponentStyles<TextActionClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<TextActionClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+    const glyphStyles: CSSRules<{}> = {
+        height: "16px",
+        width: "auto",
+        marginTop: "8px",
+        margin: "auto",
+        fill: designSystem.foregroundColor,
+    };
+
+    return {
+        textAction: {
+            position: "relative",
+            height: density(defaultHeight)(designSystem),
+            minHeight: toPx(minHeight),
+            maxHeight: toPx(maxHeight),
+            margin: `${density(designSystem.designUnit * 3)(designSystem)} 0`,
+            minWidth: "92px",
+            ...outlinePattern.rest,
+            background: ensuresBackgroundNormal,
+            borderRadius: toPx(designSystem.cornerRadius),
+            display: "flex",
+            flexDirection: "row",
+            transition: "all 0.2s ease-in-out",
+            "&:hover": {
+                ...outlinePattern.hover,
+            },
+        },
+        textAction__focus: {
+            outline: "none",
+            ...outlinePattern.focus,
+        },
+        textAction__disabled: {
+            borderColor: disabledColor(designSystem),
+            color: disabledColor(designSystem),
+            cursor: "not-allowed",
+            "&:hover": {
+                borderColor: disabledColor(designSystem),
+            },
+            "& $textAction_beforeGlyph, & $textAction_afterGlyph": {
+                opacity: ".5",
+            },
+        },
+        textAction_button: {
+            borderColor: "transparent",
+            height: `calc(${density(defaultHeight)(designSystem)} - 4px)`,
+            minHeight: toPx(minHeight),
+            maxHeight: toPx(maxHeight),
+            minWidth: "fit-content",
+            margin: "2px",
+            padding: "0 16px",
+            [applyLocalizedProperty("right", "left", direction)]: "0",
+            top: "0",
+            transition: "color .1s, background-color .1s, border-color 0.2s ease-in-out",
+            flex: "0 0 auto",
+            cursor: "pointer",
+            // Because button is passed as a rendered prop, there is no way to access internal
+            // selectors and pass style via jssStyleSheet prop
+            "& svg": {
```

Comment:
I don't think we want to target all `svg` elements here

Reply:
Yeah, this will also be removed per my comment in the meeting. Will update once the other PR is in.